go/vt/sqlparser/parse_test.go

```diff
@@ -1423,6 +1423,9 @@ var (
 		}, {
 			input:  "show session status",
 			output: "show session status",
+		}, {
+			input:  "show session status LIKE 'Ssl_cipher'",
+			output: "show session status like 'Ssl_cipher'",
```

Comment:
maybe add tests for a LIKE with a %, a naked like without quotes, and some of the where examples in docs https://dev.mysql.com/doc/refman/8.0/en/extended-show.html